geode-core/src/integrationTest/java/org/apache/geode/distributed/LocatorIntegrationTest.java

```diff
@@ -21,13 +21,11 @@
 import static org.apache.geode.distributed.ConfigurationProperties.LOCATOR_WAIT_TIME;
 import static org.apache.geode.distributed.ConfigurationProperties.LOG_FILE;
 import static org.apache.geode.distributed.ConfigurationProperties.MCAST_PORT;
+import static org.apache.geode.distributed.internal.DistributionConfig.GEMFIRE_PREFIX;
 import static org.apache.geode.internal.AvailablePort.SOCKET;
 import static org.apache.geode.internal.AvailablePort.getRandomAvailablePort;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
```

Comment:
AssertJ is a much richer assertion API that introduced to Geode a few years ago. I highly recommend using it over JUnit assertions!

Reply:
@aaronlindsey: the general preference and consensus is to use AssertJ instead of JUnit, it's been discussed some time ago in the Geode dev list: https://markmail.org/message/al3st2gko4hddwo2.